### PR TITLE
Sponsorship page

### PR DESCRIPTION
**Pull request by nailor**
Here's the sponsorship info page, plz comment

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -127,6 +127,30 @@ <h2 id="cfp">Call For Proposals</h2>
               <a href="http://goo.gl/forms/B4qzi58Lf6">Python Sverige membership signup form</a> (membership is free)
             </p>
             </p>
+            <h3>Become a sponsor</h3>
+            <p>
+              The second PyCon Sweden will be a conference with 250
```
Comment (haard) [inline index.html]: 300?

File: index.html [hunk 2]
```diff
@@ -127,6 +127,30 @@ <h2 id="cfp">Call For Proposals</h2>
...
+              attendees, all sharing the passion for Python
+              programming language. Two days of conference will be
+              filled with world class speakers, both local and
+              international, software engineers, system
```
Comment (haard) [inline index.html]: Don't forget the scientists!

File: index.html [hunk 3]
```diff
@@ -127,6 +127,30 @@ <h2 id="cfp">Call For Proposals</h2>
...
+              administrators, specialists, and other people who rely
+              on the use of Python. Conference provides an unique
```
Comment (haard) [inline index.html]: The conference provides a unique

File: sponsorship.html [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+          <li><a href="index.html#contact">Contact</a></li>
+        </ul>
+        <a href="#"><img src="assets/images/logo.png" width="200" height="56" alt="PyCon Sweden"></a>
+      </div>
+      <div>
+        <div class="row marketing">
+          <div class="col-sm-12">
+            <h2>Become a sponsor of PyCon Sweden 2015!</h2>
+            <p>
+              The second PyCon Sweden will be a conference with 250
```
Comment (haard) [inline sponsorship.html]: 300?

File: sponsorship.html [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+              attendees, all sharing the passion for Python
+              programming language. Two days of conference will be
+              filled with world class speakers, both local and
+              international, software engineers, system
+              administrators, specialists, and other people who rely
+              on the use of Python. Conference provides an unique
```
Comment (haard) [inline sponsorship.html]: The conference provides a unique

File: sponsorship.html [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+            <h2 id="cfp">Sponsorship options</h2>
+            <div class="row">
+              <div class="col-sm-4">
+                <h3>Gold sponsor (limited to 2)</h3>
+                <ul>
+                  <li>Guaranteed space for booth at the venue</li>
+                  <li>3 corporate tickets</li>
+                  <li>Full-page ad in conference brochure</li>
+                  <li>Company logo on conference bag or lanyard</li>
+                  <li>Get one rollup in each lecture hall</li>
```
Comment (haard) [inline sponsorship.html]: lecture hall, beside the stage

File: sponsorship.html [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+          <div class="col-sm-12">
+            <h2>Other sponsorship options</h2> 
+            <p>
+              If you feel that none of the above options suits you or
+              you want to contribute more, we have other options to
+              sponsor our conference.
+
+              We are always looking for sponsorships to individual
+              parts of the conference, like dinner, lunch or coffee
+              breaks. In addition we welcom travel grants if you want
```
Comment (haard) [inline sponsorship.html]: welcome

File: sponsorship.html [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+              to support people, who can't afford to travel, to attend
```
Comment (haard) [inline sponsorship.html]: ...support people who can't... (remove comma)

File: sponsorship.html [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+              discuss about other options, don't hesitate
+              to <a href="mailto:sponsorship@pycon.se">contact us</a>.
+            </p>
+          </div>
+          <div>
+            <h2>Sponsorship contract</h2>
+            <p>
+              Interested in sponsoring PyCon Sweden 2015?
+              Contact <a href="mailto:sponsorship@pycon.se">our
+              sponsorship contract Jyrki Pulliainen</a>
```
Comment (strokirk) [inline sponsorship.html]: contract -> contact

**Pull request merged**